app/models/effective/datatables/logs.rb

```diff
@@ -37,11 +55,17 @@ class Logs < Effective::Datatable
         # A nil attributes[:log_id] means give me all the top level log entries
         # If we set a log_id then it's for sub logs
         def collection
-          if attributes[:user_id].present?
-            Effective::Log.unscoped.where(:parent_id => attributes[:log_id]).where(:user_id => attributes[:user_id]).includes(:user)
-          else
-            Effective::Log.unscoped.where(:parent_id => attributes[:log_id]).includes(:user)
-          end
+          col = Effective::Log.unscoped
+            .where(parent_id: attributes[:log_id])
+            .joins('LEFT JOIN users ON users.id = logs.user_id')
```

Comment:
@jfeaver, I made this change because line 61 wouldn't work with `.includes(:user)`.

Why do you think it performs two SQL queries?
When you use `.includes()`, it helps you to solve "1+x queries" issue, but it breaks down your AR expression into 2 separate SQL queries: first - loading log records, second - loading users.
My solution helps "kill two birds with one stone" - get rid off "1+x" problem and perform only 1 query instead of 2 (logs and users load in one set of data with the help of `LEFT JOIN`), and it allows using complex `SELECT` (as in line 61 (pay attention to `USER_COLUMN_SQL` constant)).

Maybe you are worried about line 63? You think it could cause performing query second time?